bin/mochii.js

```diff
@@ -23,45 +23,16 @@ async function run(args) {
   runMochitests(args);
 }
 
-/*
- * updateArgs is a small convenience method for determining which tests to run
- * if there are no args, include all the tests
- * if there is no browser_dbg prefix, add it.
- */
-function updateArgs(args) {
-  if (args.length == 0) {
-    return ["devtools/client/debugger/new"];
-  }
-
-  const _args = args.slice(0, -1);
-  const maybeFile = args[args.length - 1];
-  const hasFile = maybeFile && !maybeFile.includes("/");
+const argString = process.argv[0].includes("bin/node")
+  ? process.argv.slice(2)
+  : process.argv;
 
-  if (!hasFile) {
-    return args;
-  }
-
-  const file = maybeFile;
-  if (file.includes("browser_dbg")) {
-    return args;
-  }
+const args = getArgs(argString);
 
-  const newFile = `browser_dbg-${file}`;
-  return [..._args, newFile];
-}
-
-if (process.mainModule.filename.includes("bin/mochii.js")) {
```

Comment:
i removed this because it was used before to differentiate btw a terminal call and an import.